community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/FunctionsAcceptanceTest.scala

```diff
@@ -208,6 +208,48 @@ class FunctionsAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerTes
     result.toList should equal(List(Map("string_numbers" -> List("1", "2", "3"))))
   }
 
+  test("properties should work on nodes") {
+
+    // When
+    val result = executeScalarWithAllPlanners[Map[String,Any]](
+      "CREATE (n:Person {name: 'Popeye', level: 9001}) RETURN properties(n) AS m"
+    )
+
+    // Then
+    result should equal(Map("name" -> "Popeye", "level" -> 9001))
+  }
+
+  test("properties should work on relationships") {
+
+    // When
+    val result = executeScalarWithAllPlanners[Map[String,Any]](
+      "CREATE ()-[r:SWOOSHES {name: 'Popeye', level: 9001}]->() RETURN properties(r) AS m"
+    )
+
+    // Then
+    result should equal(Map("name" -> "Popeye", "level" -> 9001))
+  }
+
+  test("properties should work on maps") {
+
+    // When
+    val result = executeScalarWithAllPlanners[Map[String,Any]](
+      "RETURN properties({name: 'Popeye', level: 9001})"
+    )
+
+    // Then
+    result should equal(Map("name" -> "Popeye", "level" -> 9001))
+  }
+
+  test("properties(null) should be null") {
```

Comment:
What would be sufficient, a test for every type (ignoring nested list types)?  We don't do that for other functions, in this case I felt testing lists and scalars and maps is alright but perhaps we should do more?